WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,6 +1,8 @@
 RSpec.configure do |config|
   config.before(:suite) do
+    ActiveRecord::Base.connection.execute("SET foreign_key_checks = 0;")
     DatabaseCleaner.clean_with :truncation
+    ActiveRecord::Base.connection.execute("SET foreign_key_checks = 1;")
```

Comment:
You can't truncate a table with foreign key constraints. Setting `foreign_key_checks` to 0 allows you to truncate the tables (bypassing any foreign key checks) and then setting the value back to 1 re-enables this.